### PR TITLE
WIP add the possibility to manage (or not) aliases

### LINKED ISSUE #237: Add possibility to manage (or not) /etc/aliases
Add possibility to manage (or not) /etc/aliases with the module.

### PR DESCRIPTION
**Pull request by your1p**
fix #237

### CODE REVIEW

File: manifests/files.pp [hunk 1]
```diff
@@ -43,12 +44,18 @@
   }
 
   # Aliases
-  file { '/etc/aliases':
-    ensure  => 'file',
-    content => "# file managed by puppet\n",
-    notify  => Exec['newaliases'],
-    replace => false,
-    seltype => $postfix::params::aliasesseltype,
+  case $manage_aliases {
```
Comment (raphink) [inline manifests/files.pp]: How about `if $manage_aliases` instead of a `case`?

File: manifests/files.pp [hunk 2]
```diff
@@ -43,12 +44,18 @@
   }
 
   # Aliases
-  file { '/etc/aliases':
-    ensure  => 'file',
-    content => "# file managed by puppet\n",
-    notify  => Exec['newaliases'],
-    replace => false,
-    seltype => $postfix::params::aliasesseltype,
+  if $manage_aliases {
```
Comment (raphink) [inline manifests/files.pp]: Thanks for switching to `if`. However, this means the `true` and `default` sections need to be removed.

Comment (your1p) [reply manifests/files.pp]: I am so sorry. I pushed the wrong commit. It is fixed now.

Comment (your1p) [reply manifests/files.pp]: I've pushed the write branch.

File: manifests/files.pp [hunk 3]
```diff
@@ -45,18 +45,17 @@
...
-    default: {
+    file { '/etc/aliases':
+      ensure  => 'file',
+      content => "# file managed by puppet\n",
+      notify  => Exec['newaliases'],
+      replace => false,
+      seltype => $postfix::params::aliasesseltype,
     }
   }
+  else {
```
Comment (raphink) [inline manifests/files.pp]: What is the point of keeping an empty `else`  section?

**Comment by Bodenhaltung** [comment]
Hmm, this PR is dead?

**Comment by raphink** [comment]
Fixed in #271

**Pull request closed without merging**